### DISCUSSION TITLE
如何对深度学习训练进行性能调优？ | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#12] [General]
# 如何对深度学习训练进行性能调优？ | 高策

江湖小虾米

http://gaocegege.com/Blog/kubernetes/metrics-survey

**Comment by Marsrocky** [comment]
学到了学到了，定期来学习gaoce哥哥的blog！

**Reply by gaocegege** [reply]
doge

**Comment by Rouzip** [comment]
CPU profile已经被扔进垃圾箱了orz
